Add `onSelectedCellChange` event (#3328)
adazzle/react-data-grid: README.md

###### `onCellKeyDown?: Maybe<(args: CellKeyDownArgs<R, SR>, event: CellKeyboardEvent) => void>`

###### `onSelectedCellChange?: Maybe<(args: CellSelectArgs<R, SR>) => void>;`

Triggered when the selected cell is changed.

Arguments:

> - `args.idx`: `number` - column index

Author: Oops... I forgot to remove it. Thanks.

> ###### `onCellSelected?: Maybe<(args: CellSelectArgs<R, SR>) => void>;`
> 
> Triggered when a cell is selected by click, double click, right click, Page{Up|Down|Left|Right} key, Tab key, or any other actions.

Reviewer: ```suggestion
###### `onSelectedCellChange?: Maybe<(args: CellSelectArgs<R, SR>) => void>;`

Triggered when the selected cell is changed.
```
